modules/33_CDR/portfolio/declarations.gms

```diff
@@ -9,7 +9,7 @@ scalars
 s33_capture_rate            "CO2 capture rate for capturing emissions, e.g., from burning natural gas" / 0.9 /
 
 s33_co2_rem_pot             "specific carbon removal potential [Gt C per Gt ground rock]"
-s33_co2_rem_rate            "carbon removal rate [fraction of annual reduction of total carbon removal potential]"
+s33_co2_rem_fraction        "fraction of stone weathering per year [fraction of annual reduction of total carbon removal potential], independent of climate grade"
```

Comment:
@strefler I find the description still hard to interpret and don't really understand the rationale behind this scalar. Why would it reduce the total carbon removal potential annually?

Reply:
In my understanding it follows from the  calculation of the removal per time step:
rocks_on_field * share_weathering * cum_removal_potential 
So the share weathering is equal to the reduction of the remaining removal potential. But I agree that it is confusing, and would be in favor or removing it.

Reply:
I looked it up in Jess paper. 
Our friend `s33_co2_rem_rate` is d(x) from Equation (1) in the [paper](https://iopscience.iop.org/article/10.1088/1748-9326/aaa9c4/pdf). That is **the weathering rate at ambient temperature (25°C)**. But I don't know what that means in terms of humidity. Maybe there is an established standard in this field of science.
In any case, [in the SI](https://content.cld.iop.org/journals/1748-9326/13/3/034010/revision2/ERL_034010_SD.pdf?Expires=1734966574&Signature=ddFnWUZqCnXpE5HLHikPUOvNPLNtLM1fbQidbIkDySR6j~YCetjfM74G-KvOl2xsxUVnriVGMTXUyYHkvNOgwj7ihT879osBGZxCoUqm1Pq9NLibXz9y6IN-KlIGPviYJj0x6exijz9-39h6OqIdOiXWR0iNdDy9TpPgcKOUDAOCYGvgJfy1HAMJvA71LTA3ooLblU01bqHgTDVl7oUweW3gXUbxJFieUHXo9hDm8h-s8ZqKYYXlpkzQn-gPzYfSzHCYrEOi5esvJuTIlnLAojgpEjIPkOIM~N7tC0drApL9Qi3ekZxUeLVipYt3LlO4P8Ai3BVtB1o-CaXafjCaGA__&Key-Pair-Id=KL1D8TIY3N7T8), there is a table with weathering rate correction factors to account for temperature deviations from the ambient case in Eq.1.
And these numbers are also multiplied to get from s33_co2_rem_rate to p33_co2_rem_rate.
But some of the calculation on how to calculate the weathered rocks is also hidden in the calculation of p33_co2_rem_rate and cannot be understood by looking at only either the datainput or the equations.
Is there a reason to have the logarithm in the calculation of p33_co2_rem_rate and then revert that kinda in the equations.gms with the exponent? Yes it shortens the code in equations-gms, but it is less transparent and also naming p33_co2_rem_rate becomes quite tricky.